sigh/lib/sigh/resign.rb

```diff
@@ -121,7 +124,7 @@ def installed_identies
       ids = []
       available.split("\n").each do |current|
         begin
-          (ids << current.match(/.*\"(.*)\"/)[1])
+          (ids << "#{current.match(/[a-zA-Z0-9]{40}/)}#{ID_SEPARATOR}#{current.match(/.*\"(.*)\"/)[1]}")
```

Comment:
@asfalcone I suppose that solution from @baarde looks much better
It might be better to recreate PR by @baarde and accept it, and my PR should be closed.
@asfalcone what do you think?